source/transfer_list.rst

```diff
@@ -1032,6 +1032,44 @@ into its memory map during platform setup. If other memory types are required
      - hdr_size + 0x8
      - The size of the memory region.
 
+**DT formatted FF-A manifest entry layout (XFERLIST_DT_FFA_MANIFEST)**
+
+This entry type holds the FF-A manifest image whice is in DT format [DT]_,
+as described in [TFAFFAMB]_.
+This manifest contains the SP (Secure Partition) configuration, consumed
+by the SPMC at boot time.
+
+It may also contain some information to the SP itself.
+
+.. _tab_dt_ffa_manifest:
+.. list-table:: DT formatted FF-A manifest type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **0x106**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size of FF-A manifest in bytes.
+
+   * - ffa_manifest
+     - data_size
+     - hdr_size
+     - Holds a FF-A manifest image in DT format.
```

Comment:
DT might be better as devicetree? But so long as there is DT elsewhere, this is fine.

Is this format per the link here?:

https://hafnium.readthedocs.io/en/latest/secure-partition-manager/secure-partition-manager.html#spmc-manifest

If so, can you please mention it here, with a reference, or similar.

Also, that link seems to imply that the DT is partially the same as normal DT schema, but with some additions. I see an 'attribute' node with no compatible string and property names which use underscore when they should use hyphen. Also, some mention should be made there of the address-cells property being 2?

You might also be interested in the Universal Payload project:

https://universalpayload.github.io/spec/index.html

Reply:
Hi @sjg20. The DT acronym is defined earlier, although more the doc generally could be more consistent on its use of acronyms (that doesn't need to be fixed here)
This TE follows the format in https://trustedfirmware-a.readthedocs.io/en/latest/components/ffa-manifest-binding.html, which is already referenced.
Your feedback on those reference docs is useful, thanks.
Finally we should discuss the UP project sometime but that's beyond the scope of this PR.

So I think we're good to go?